# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../bincimap/uninitialized-fix.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- bincimap-1.3.4/src/mime-parsefull.cc.orig	2006-01-04 22:50:25.000000000 +0100
+++ bincimap-1.3.4/src/mime-parsefull.cc	2006-01-04 22:53:31.000000000 +0100
@@ -343,14 +343,14 @@
 
   // Read two more characters. This may be CRLF, it may be "--" and
   // it may be any other two characters.
-  char a;
+  char a = 0;
   if (!mimeSource->getChar(&a))
     *eof = true;
 
   if (a == '\n')
     ++*nlines; 
 
-  char b;
+  char b = 0;
   if (!mimeSource->getChar(&b))
     *eof = true;
     
@@ -560,13 +560,13 @@
   delete [] boundaryqueue;
 
   if (toboundary != "") {
-    char a;
+    char a = 0;
     if (!mimeSource->getChar(&a))
       *eof = true;
 
     if (a == '\n')
       ++*nlines;
-    char b;
+    char b = 0;
     if (!mimeSource->getChar(&b))
       *eof = true;
 
